src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -672,7 +672,7 @@ final boolean isLoggingPackets() {
     int numMsgsSent = 0;
     int numMsgsRcvd = 0;
 
-    private final transient Lock lock = new ReentrantLock();
+    private final transient Lock TDSChannelLock = new ReentrantLock();
```

Comment:
Minor/Optional suggestion:
Aren't local variables supposed to start with a lowercase letter?
```suggestion
    private final transient Lock tdsChannelLock = new ReentrantLock();
```

Reply:
I thought since TDS was an acronym, not a word, we preserve the case for clarity. But looking over again, this is done nowhere else, so yes, it should be lowercase.